DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/DelegateGroupsControllerTests.cs

```diff
@@ -455,5 +455,69 @@ public void EditDelegatesGroupDescription_should_redirect_to_index_action()
 
             result.Should().BeRedirectToActionResult().WithActionName("Index");
         }
+
+        [Test]
+        public void EditGroupName_should_redirect_to_index_action_when_update_is_successful()
+        {
+            // Given
+            const int groupId = 103;
+            const int centreId = 2;
+            var model = new EditGroupNameViewModel()

```

Comment:
Formatter should probably be removing the brackets here.

Reply:
sorry, I ran the formatter